networkx/algorithms/boundary.py

```diff
@@ -59,6 +59,20 @@ def edge_boundary(G, nbunch1, nbunch2=None, data=False, keys=False, default=None
         are specified and `G` is a multigraph, then edges are returned
         with keys and/or data, as in :meth:`MultiGraph.edges`.
 
+    Examples
+    -------
+    >>> G = nx.gnp_random_graph(10, 0.3, seed = 100)
```

Comment:
Can you make this example use a graph where the structure of the graph is known and easy to see what the answer should be?  Using a random graph as an example makes it hard to tell what the function is doing -- because we don't know what the graph is.

Maybe a ladder graph would work with nbunch1 being the nodes on one side and nbunch2 being either `None` (meaning all the nodes on the other side of the ladder) or a list of some of the nodes on the other side of the ladder. Make the ladder small so it's easy to follow.  Or I'm sure there are other example graphs that could be constructed -- some of which might be better examples.

Reply:
Got it, I changed the example to use a wheel_graph with 6 nodes instead.